### ISSUE TITLE
Possible NullReferenceExceptions in NpcDialog.cs

### ISSUE THREAD

**Issue by ljars** [suriyun-production/mmorpg-kit-docs#1566]
In version 1.75c, in NpcDialog.RenderUI(), lines 137-138 there is the following code:
```
if (uiNpcDialog.onSwitchToNormalDialog == null)
    uiNpcDialog.onSwitchToNormalDialog.Invoke();
```
This would cause an exception if the event is null, and the event wouldn't be invoked otherwise.
Was this condition meant to be `!=`?
There is similar code on lines 152-153, 193-194, and 207-208.

**Comment by insthync** [comment]
I put wrong condition, I will fix it.

**Comment by insthync** [comment]
Fixed in 1.75d
